src/json-schema-version.cpp.in

```diff
@@ -0,0 +1,44 @@
+#include <nlohmann/json-schema-version.hpp>
+
+namespace nlohmann
+{
+namespace json_schema
+{
+namespace about
+{
+
+const std::string &version_txt()
+{
+	// clang-format off
```

Comment:
why clang-format off? Does it break things?

Reply:
> The version.[ch]pp.in -files could be completely generic and with two or three defines (or only one? `JSON_SCHEMA_VALIDATOR_API`) outside their inclusion could be completely independent.

I don't get it. What you mean with "generic"?
The down-side of defines is, that they are potentially subject of name clashes since they are always in public scope. We use the approach of `<SOME-NS>::about>` to have the version info sorted by namespace and libraries. That is our interpretation of generic.